### ISSUE TITLE
decorative icons should have `aria-hidden="trur"`

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#3242]
### Description of the bug

When using VoiceOver on our storefront i noticed that some of the decorative svgs was read as "bilde" and nothing more. I tested using `aria-hidden="true"` and VoiceOver ignored them. 

Here are some examples: 
The "diamond" icons inside the cards
![Image](https://github.com/user-attachments/assets/0abc157c-2845-4df7-b595-62f61384b6dc)

![Image](https://github.com/user-attachments/assets/b9abead8-f214-40a9-9fae-ffccc29d5993)

![Image](https://github.com/user-attachments/assets/d654e5c9-1aed-48a6-9c5d-2574d20f358d)

### Steps To Reproduce

I recommend using Siteimproves Chrome extension when testing for this. Wave do not create a warning or error for this issue. 

Just go to a page and run site improve. It will give you an error where svgs dont have a text. 

![Image](https://github.com/user-attachments/assets/4d315b43-c080-4e2c-b444-29eb881ec1b6)
